features/fixtures/ast.yaml

```diff
@@ -1,87 +1,112 @@
-_version: 2.1
-metadata:
-- name: "Format"
-  value: "1A"
+
+_version: "2.1"
```

Comment:
I think this is an inconsistency problem. In `ast.json`, we are serializing version as string.

```json
{
  "_version": "2.1"
}
```

But in `ast.yaml`, we are serializing this as a number (or a string without quotes)

```yaml
_version: 2.1
```

While building the object using libsos, I made it a string which is why you see the above. I think we need to decide on how to do this.